docs/changes/coordinates/14231.feature.rst

```diff
@@ -0,0 +1,2 @@
+The ``oldstyle`` option is now available for ``to_string``.
```

Comment:
Maybe for once decisions have been made, but I think this should be an `api` that states that spaces are now inserted before units in `Angle.to_string()`.